src/App.tsx

```diff
@@ -17,18 +38,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                value={query}
+                handleInputQuery={setQuery}
+                onSelectedCategory={setSelectedCategory}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {isLoading
+                ? <Loader />
+                : (
+                  <TodoList
+                    todos={preparedTodos}
+                    onSelectedUser={setSelectedTodo}
+                    onSelectedTodo={selectedTodo}
```

Comment:
I think it would be better to name it `selectedTodo={selectedTodo}` because with `onSelectedTodo` it seems like we expect some function there instead of some value

Reply:
i think you used that way of naming somewhere else in your code btw

Reply:
Good point, fix this